### ISSUE TITLE
permissions unreliable to remove, persist in policy

### ISSUE THREAD

**Issue by veyga** [propensive/fury-old#1389]
Related to #776 

'fury obviate xxx' does not reliably recognize the permission hash. 

Hash | Class | Target | Action
------- | ------- | --------- | --------
865 | java.net.SocketPermission | localhost:666 | accept,connect,listen
87c | java.net.SocketPermission | localhost:667 | accept,connect,listen

> fury permission obviate -permission 865 
The permission 865 was not found.
> fury permission obviate -P 87c
The permission 87c was not found.

The policy file also does not seem to correctly account for the scope input from the cli. Both directory and layer scopes look to resolve the same, so there is a disconnect from the fury layer, and the permissions attributed to that particular layer.

**Comment by propensive** [comment]
Yes, there's definitely some work required here. At least part of it is a UI issue: `require` and `obviate` affect the layer definition (while `grant` automatically grants the permission), and `grant` and `revoke` don't change the layer, but affect the system. To complicate matters, permissions are inherited from dependencies, so if you depend on a module which needs a permission, you need to grant that permission. But if you want to obviate a permission, it may originate from a dependency instead of the module you are editing, in which case it won't be found.

The scopes are not fully implemented.
